src/main.rs

```diff
@@ -62,9 +63,10 @@ pub async fn main() -> Result<(), anyhow::Error> {
                 .requires("bindle"),
         )
         .arg(
-            Arg::with_name("BINDLE_URL")
+            Arg::with_name(BINDLE_URL)
                 .long("bindle-server")
-                .value_name("BINDLE_URL")
+                .value_name(BINDLE_URL)
```

Comment:
Not blocking, but do we want to have consistency between the flag and environment variable name?

Reply:
I'm ok without it because variable names generally need to be prefaced with the binary name, whereas in context of the command that is just overly verbose

Reply:
Good point. I will change that before merging.